src/App.jsx

```diff
@@ -1,9 +1,38 @@
 import './App.css';
+import soldTV from "./helpers/soldTV.js";
+import {inventory} from "./constants/inventory.js";
 
```

Comment:
importeer ook de overige 2 helper functies, dat is alleen mogelijk wanneer deze geëxporteerd worden.